arm_wrapper.sh

```diff
@@ -1,2 +1,4 @@
 #!/bin/bash
-echo bash /opt/arm/identify.sh /opt/arm/config | at now
+DEVNAME=$1
+udevadm info --query=env "/dev/$DEVNAME" > /tmp/arm_disc_info
```

Comment:
This has the potential to cause a race condition for people ripping with multiple optical drives ripping things in parallel.  Can you change the output to something like /tmp/arm_disc_info_$DEVNAME ?